src/test/java/io/tiledb/java/api/DatatypeTest.java

```diff
@@ -288,4 +288,13 @@ public void testIsRealDtypes() {
       }
     }
   }
+
+  @Test
+  public void testSize() throws TileDBError {
+    Assert.assertEquals(4, Datatype.TILEDB_FLOAT32.size());
```

Comment:
Not sure how these were selected, but if we're using it as a spot-check, might also check a {U}INT16.

Reply:
added them all!